README.md

```diff
@@ -312,7 +312,7 @@ Example:
 ```
 -e VALHEIM_PLUS=true \
 -e VPCFG_Server_enabled=true \
--e VPCFG_Server_enforceMod=false \
```

Comment:
This is wrong. For changing the datarate enforceMod must not be true.

Reply:
This line is in the Translation table section and has nothing to do with datarate.